### PR TITLE
Automate container creation using zuul

### PR DESCRIPTION
**Pull request by JuanPTM**

### CODE REVIEW

File: scripts/push.sh [hunk 1]
```diff
@@ -19,12 +19,12 @@ docker tag "$REPOSITORY:$REVISION" "$REPOSITORY:$VERSION"
 
 # TODO: this really needs review
 # push e.g. scs/keycloak:19.0.3
-if [[ $IMAGE == "keycloak" ]]; then
+if [[ $IMAGE == "scs-keycloak" ]]; then
```
Comment (berendt) [inline scripts/push.sh:22]: Dito.

File: scripts/push.sh [hunk 2]
```diff
@@ -19,12 +19,12 @@ docker tag "$REPOSITORY:$REVISION" "$REPOSITORY:$VERSION"
...
     version=$(docker run --rm "$REPOSITORY:$VERSION" keycloak --version | awk '{ print $2 }')
-    if skopeo inspect --creds "${DOCKER_USERNAME}:${DOCKER_PASSWORD}" "docker://${REPOSITORY}:${version}" > /dev/null; then
-        echo "The image ${REPOSITORY}:${version} already exists."
+    if skopeo inspect --creds "${DOCKER_USERNAME}:${DOCKER_PASSWORD}" "docker://${REPOSITORY}:${VERSION}" > /dev/null; then
+        echo "The image ${REPOSITORY}:${VERSION} already exists."
```
Comment (berendt) [inline scripts/push.sh:24]: The script currently reads the version from the image itself and sets it. Hence version and not VERSION. If you want to change this you also have to remove line 23.

File: scripts/push.sh [hunk 3]
```diff
@@ -19,12 +19,12 @@ docker tag "$REPOSITORY:$REVISION" "$REPOSITORY:$VERSION"
...
     else
-        docker tag "$REPOSITORY:$REVISION" "$REPOSITORY:$version"
-        docker push "$REPOSITORY:$version"
+        docker tag "$REPOSITORY:$REVISION" "$REPOSITORY:$VERSION"
+        docker push "$REPOSITORY:$VERSION"
```
Comment (berendt) [inline scripts/push.sh:26]: Dito.

File: .github/workflows/build-keycloak-container-image.yml [hunk 4]
```diff
@@ -43,17 +43,16 @@ jobs:
...
       - name: Push container image
-        run: |
-          scripts/push.sh
+        run: scripts/push.sh
         env:
           DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
-          IMAGE: keycloak
-          REPOSITORY: scs/keycloak
+          IMAGE: scs-keycloak
+          REPOSITORY: scs-container-images/scs-keycloak
```
Comment (berendt) [inline .github/workflows/build-keycloak-container-image.yml:55]: The fact that it is an SCS image results from the fact that the image is in an SCS namespace? I would prefer to keep keycloak as the name. I think it is better to add a label if this is to be listed explicitly.

File: scripts/push.sh (hunk 2 above)
Comment (JuanPTM) [reply scripts/push.sh:24]: Removed line 23, keep the VERSION tag.

File: scripts/push.sh (hunk 3 above)
Comment (JuanPTM) [reply scripts/push.sh:26]: removed the re tag

File: .github/workflows/build-keycloak-container-image.yml (hunk 4 above)
Comment (JuanPTM) [reply .github/workflows/build-keycloak-container-image.yml:55]: As discussed on the weekly IAM meeting of kw3, nobody veto the `scs-keycloak` name, so we keep it to differentiate it from a base Keycloak image as we have plugings already instaled.

**Review by reqa — APPROVED** [review]
LGTM

**Pull request merged**